custom/icds_reports/models/aggregate.py

```diff
@@ -240,6 +240,10 @@ class AwcLocation(models.Model, AggregateMixin):
     block_is_test = models.SmallIntegerField(blank=True, null=True)
     supervisor_is_test = models.SmallIntegerField(blank=True, null=True)
     awc_is_test = models.SmallIntegerField(blank=True, null=True)
+    awc_deprecated_at = models.DateField(blank=True, null=True)
```

Comment:
assume it's intentional that these are not `datetime`s?

Reply:
Yeah, time does not matter for these and I thought that dates would make comparisons easier

Reply:
just thinking if these should be kept as `datetime` as well so mirror what HQ is keeping. I understand the time is kind of irrelevant for dashboard but if it does not hurt in terms of performance etc why not keep it the same as other places?

Reply:
The dashboard only cares about days, and in this case, only cares about months (date with day of 1, so 2020-03-01). That means this will only be compared to dates and we want it to always be treated as the date that it represents, or a month truncated version, not a datetime. I think leaving it as a datetime in the database opens us up to a number of comparison bugs (`2020-03-01 01:04 > 2020-03-01`) that could otherwise be avoided by making sure everything is a date to begin with, rather than having to remember to coerce it each time

Reply:
cool. thanks for explaining this @calellowitz